authd.proto

```diff
@@ -45,6 +45,7 @@ message SBRequest {
   string broker_id = 1;
   string username = 2;
   string lang = 3;
+  string mode = 4;
```

Comment:
Maybe this is one place where we can use an enum, wdyt?

Reply:
Sounds good to me. This way, we kind of avoid the whole "not defined" issue.